pagopa/io-app
# chore: [IOPAE-1120] Add Institution Services screen

## Short description
This PR adds the `InstitutionServicesScreen` to handle the institution's services

<details open><summary>Details</summary>
<p>

| institution  |
| - |
| <video src="https://github.com/pagopa/io-app/assets/29163287/6f6474e8-e2ea-4df9-8aeb-e1f4ff2765b3" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- added `InstitutionServicesScreen` component
- added `ServicesHeaderSection` component (shared between `InstitutionServicesScreen` and `ServiceDetailsScreen`)
- added `ServicesHeader` component
- renamed `handleFindServices` to `handleFindInstitutionServices`  
- removed `ServicesWebviewScreen`
- added tests
- updated locales

## How to test
Using `io-dev-api-server`, navigate to the services tab and tap an institution.Check that the screen is displayed correctly.
